docs/operator-manual/user-management/microsoft.md

```diff
@@ -92,107 +80,125 @@
+      config:
+        clientID: $MICROSOFT_APPLICATION_ID
+        clientSecret: $MICROSOFT_CLIENT_SECRET
+        redirectURI: http://localhost:8080/api/dex/callback
+        tenant: ffffffff-ffff-ffff-ffff-ffffffffffff
+        groups:
+          - argocd_access_SG
+          - argocd_org_admin_SG
+          - argocd_admin_SG
+```
 
-            policy.default: role:readonly
-            policy.csv: |
-               p, role:org-admin, applications, *, */*, allow
-               p, role:org-admin, clusters, get, *, allow
-               p, role:org-admin, repositories, get, *, allow
-               p, role:org-admin, repositories, create, *, allow
-               p, role:org-admin, repositories, update, *, allow
-               p, role:org-admin, repositories, delete, *, allow
-               g, "84ce98d1-e359-4f3b-85af-985b458de3c6", role:org-admin
+## Configuring authorization through RBAC
+Edit the `argocd-rbac-cm` ConfigMap to assign permissions from the user's Azure AD security group memberships. After being logged in into ArgoCD, check [the user's group membership](log-in-to-argocd-ui-using-sso) to decide if you have to use the security group's `Object ID` or `name`.
 
-4. Mapping role from jwt token to argo
-   If you want to map the roles from the jwt token to match the default roles (readonly and admin) then you must change the scope variable in the rbac-configmap.
+- [RBAC Configurations](../rbac.md) contains more detailed scenarios.
+- [operator-manual/argocd-rbac-cm.yaml](https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-rbac-cm.yaml) contains available variables.
 
-            policy.default: role:readonly
-            policy.csv: |
-               p, role:org-admin, applications, *, */*, allow
-               p, role:org-admin, clusters, get, *, allow
-               p, role:org-admin, repositories, get, *, allow
-               p, role:org-admin, repositories, create, *, allow
-               p, role:org-admin, repositories, update, *, allow
-               p, role:org-admin, repositories, delete, *, allow
-               g, "84ce98d1-e359-4f3b-85af-985b458de3c6", role:org-admin
-               scopes: '[roles, email]'
+### Dedicated security group for administrators
+To map a special security group to the ArgoCD `admin` role, you can use the following `argocd-rbac-cm` ConfigMap:
 
-   Refer to [operator-manual/argocd-rbac-cm.yaml](https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-rbac-cm.yaml) for all of the available variables.
+```yaml
+data:
+  policy.default: role:readonly
+  policy.csv: |
+    # either use the security group name
+    g, "argocd_admin_SG", role:admin
+    # or the group Object ID
+    g, "84ce98d1-e359-4f3b-85af-985b458de3c6", role:admin
+```
 
-## Azure AD App Registration Auth using Dex
+### Detailled permission
+This `argocd-rbac-cm` ConfigMap maps the Azure AD security group `argocd_org_admin_SG` to the restricted ArgoCD role `org-admin`:
```

Comment:
s/ArgoCD/ArgoCD

Reply:
Fixed (and other occurences without whitespace)